site/en/docs/web-platform/chrome-variations/index.md

```diff
@@ -0,0 +1,171 @@
+always predict how real users will respond to changes and new features, at scale.  
+
+Chrome Variations is a crucial mechanism to help with this, making it possible to check usage and
+metrics for Chrome Stable with a feature either activated or deactivated. By measuring the impact of
+changes in Chrome Stable, we can ship the best features possible and build a better browser—even
+when the trade-offs are complex.
+
+## When do Chrome engineers use Chrome Variations?
+
+There are three main reasons that Chrome needs Chrome Variations.
+
+### Activate a new feature 
+
+Using Chrome Variations to provide control over activation of a new feature is particularly useful
+for anything that might be more risky in some way, or that might affect performance.
+
+Chrome Variations can enable a new feature to be rolled out to a subset of users, and then Chrome
+engineers can check for differences in performance or look at other types of feedback from the
+variation group. 
+
+### Deactivate a feature
+
+Chrome Variations can be used as a safety mechanism to turn off a feature—although this only happens
+rarely. For example, a new networking feature might become subject to a denial-of-service attack.
+Chrome Variations can be used to quickly turn off a feature like that, since the Chrome Variations
+configuration is downloaded every 30 minutes, and settings are activated every time you restart
+Chrome. By contrast, simply updating Chrome and waiting for the new version to be propagated to
+billions of users would be much slower.
+
+### Try changes to a feature
+
+Lastly, Chrome Variations can be used to validate changes and updates. For example, we can try
+tweaking the number of results shown in the
+[Omnibox](https://blog.chromium.org/2021/11/searching-browsing-shutdown-chrome-performance-improvements.html).
+
+Chrome Variations can also be used to study the long-term effects of new features. This is done by
+using the Chrome Variations mechanism to hold back features for a small proportion of users, maybe
+1% or less. A **holdback group** can be useful to check for changes and side effects that only
+become visible over time.
+
+A holdback group can be particularly important for user interface changes, where users are likely to
+interact with a feature when it's new, but may behave differently in the long term. For example,
+photo picker functionality was launched in Chrome, but the feature was temporarily held back from a
+small percentage of users, to see how outcomes compared. We shipped the new Android Photo Picker to
+a high proportion of our users, and initially saw a significant increase in the number of images
+that group shared on the Web. However, in the six months that we had the holdback variation, we saw
+the lift in usage decrease significantly. This was because availability of the new photo picker
+encouraged sites to add the accept attribute to file input elements—which led to a better experience
+for all users.
```

Comment:
```suggestion
A holdback group can be particularly important for user interface changes, where users are likely to
interact with a feature when it's new, but may behave differently in the long term. 

For example, we launched a photo picker functionality in Chrome, but the feature was temporarily held
back from a small percentage of users, to compare outcomes. We shipped the new Android Photo Picker to
a high proportion of our users, and initially saw a significant increase in the number of images
that group shared on the Web. However, in the six months that we had the holdback variation, we saw
the lift in usage decrease significantly. This was because availability of the new photo picker
encouraged sites to add the accept attribute to file input elements—which led to a better experience
for all users.
```

Reply:
Mostly this suggestion is to add extra space to a very long paragraph.

Reply:
👍